### PR TITLE
Fixed issue with parsing via trigger source with leading comments

### PR DESCRIPTION
**Pull request by jgebal**
Resolves problem when sources are passed via JDBC or dynamic SQL for compilation and source code contains leading comments.
The code was adjusted to properly read sources even when they are getting passed with leading comments containing create package statement.
Now, the following will be properly parsed:
```sql
  begin
    execute immediate q'[
      /*
      * Some multiline comments before package spec
      create or replace package dummy_test_package dummy comment to prove that we pick the right piece of code
      */
      -- create or replace package dummy_test_package dummy comment to prove that we pick the right piece of code
      --Some single-line comment before package spec
      create or replace package dummy_test_package as
        --%suite(dummy_test_suite)
        --%rollback(manual)

        --create or replace package dummy_test_package dummy comment to prove that we pick the right piece of code
        
        --%test(dummy_test)
        --%beforetest(some_procedure)
        procedure some_dummy_test_procedure;
      end;]';
  end;
/
```

### CODE REVIEW

File: source/core/annotations/ut_annotation_manager.pkb [hunk 1]
```diff
@@ -267,10 +267,11 @@ create or replace package body ut_annotation_manager as
       l_sql_text    ora_name_list_t := a_sql_text;
```
Comment (lwasylow) [inline source/core/annotations/ut_annotation_manager.pkb]: Is there a need for that variable anymore its passed as attribute anyway.

**Comment by lwasylow** [comment]
In build there is some odd log out during installation :
Synonym UT_RUN_INFO created.
bash: del: command not found
Installing component UT_TRIGGER_ANNOTATION_PARSING

File: source/core/annotations/ut_annotation_manager.pkb (hunk 1 above)
Comment (jgebal) [reply source/core/annotations/ut_annotation_manager.pkb]: Good spot - fixed

**Comment by jgebal** [comment]
The message comes from script: `define_ut3_owner_param.sql` I'm not sure how else to remove temp files using sqlpls for both Win and Unix/Mac

**Pull request merged**